whisper/whisperv6/doc.go

```diff
@@ -57,6 +56,7 @@ const (
 	aesKeyLength    = 32
 	AESNonceLength  = 12
 	keyIdSize       = 32
+	bloomFilterSize = 64
```

Comment:
nitpick: I would like to see a unit for that "size" Bytes? Bits? String length? A comment like the one when you define `DefaultSyncAllowance` below.